test/test_integration_credentials.py

```diff
@@ -32,25 +35,21 @@ def setup_cli_params(self, env_vars, *params):
     def setUp(self):
         self.setup_cli_params({})
         try:
+            utils.delete_credentials()
             utils.remove_file(DEFAULT_NETRC_FILE)
```

Comment:
this shouldnt be here because we arent working with netrc. We should have the keyring equivalent `delete_password`